src/store/services/mutations.ts

```diff
@@ -9,8 +10,7 @@ const { commit } = getStoreAccessors<ServicesState, RootState>('services');
 const mutations = {
 
   addService(state: ServicesState, service: Service) {
-    state.allIds.push(service.id);
-    state.byId[service.id] = { ...service };
+    Vue.set(state.services, service.id, { ...service });
```

Comment:
you're removing `allIds`, but that's meant for easy home keeping, You can remove an id from this list and it will not be displayed anymore since you can easily map over it. I think you favor inspecting the object's keys, that okay too, but I want you to know the reason behind this approach.